### DISCUSSION TITLE
Support MySQL Datasource

### DISCUSSION THREAD

**Discussion by teckick** [databendlabs/databend#4022] [Ideas]
Hi!

Is there a plan to support MySQL datasource and query on multiple datasource? (like ClickHouse or Presto)

**Comment by wubx** [comment]
It's good idea.

**Comment by bohutang** [comment]
Good idea.
As one author of ClickHouse MaterialMySQL engine, to support it we need to do a lot of work, so there is no priority plan on it.
Maybe we would add MySQL as one of our ELT(This has not start) data source not Databend data source then.

**Comment by wubx** [comment]
I think he just want datasource connect  engine
https://clickhouse.com/docs/en/engines/database-engines/mysql/
